src/common/lib/utils.js

```diff
@@ -45,6 +45,11 @@ const utils = {
     }
     return hex.join("");
   },
+  buf2hex: (buffer) => {
```

Comment:
I think you can use the `bytesToHexString` which we have a few lines below?

Reply:
Correct, if I turn the buffer into a Uint8Array.

Reply:
ah thought it was using the same, didn't look close enough.